docs/couch_to_sql_models.rst

```diff
@@ -67,7 +67,7 @@ This should contain:
     from corehq.apps.hqadmin.models import HqDeploy
     attrs = ['environment', 'user', 'date']
     docs = list(get_all_docs_with_doc_types(HqDeploy.get_db(), ['HqDeploy']))
-    [(attr, len([doc for doc in docs if not doc.get(attr)]), max([len(doc.get(attr, '')) for doc in docs])) for attr in attrs]
+    [(attr, len([doc for doc in docs if not doc.get(attr)]), max([len(doc.get(attr)) for doc in docs if doc.get(attr)] + [0])) for attr in attrs]
```

Comment:
I'm worried about situations where there are A LOT of docs for a doc type.

I don't know if that's something we need to worry about, but if it is, should we flip this around so we iterate `docs` once in an outer loop, and iterate `attrs` in the inner loop? 

Maybe something like
```suggestion
    docs = get_all_docs_with_doc_types(HqDeploy.get_db(), ['HqDeploy'])

    num_blank = defaultdict(lambda: 0)  # Assuming you want None and ""
    max_len = defaultdict(lambda: 0)
    for doc in docs:
        for attr in attrs:
            if doc.get(attr):
                max_len[attr] = max(len(doc["attr"]), max_len[attr])
            else:
                num_blank[attr] += 1

    [(attr, num_blank[attr], max_len[attr]) for attr in attrs]
```

MUCH uglier than yours, but only keeps 100 (`chunk_size`) docs in memory, and only fetches them once.

Reply:
that's great, I'm going to merge this PR but look at putting that code into a management command and referencing it here